recipes-kernel/linux/linux-yocto_5.10.bbappend

```diff
@@ -0,0 +1,40 @@
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+B = "${WORKDIR}/build"
+
+# Support to build dtbo for RZ SBC
+KERNEL_DTC_FLAGS = "-@"
+KERNEL_DEVICETREE_OVERLAY ?= ""
+
+do_compile:prepend() {
+    if [ -n "${KERNEL_DTC_FLAGS}" ]; then
+       export DTC_FLAGS="${KERNEL_DTC_FLAGS}"
+    fi
+}
+
+# Append steps to build the device tree overlays (dtbo)
+do_compile:append() {
+    for dtbf in ${KERNEL_DEVICETREE_OVERLAY}; do
+        dtb=`normalize_dtb "$dtbf"`
+        oe_runmake $dtb CC="${KERNEL_CC} $cc_extra " LD="${KERNEL_LD}" ${KERNEL_EXTRA_ARGS}
+    done
+}
+
+# Install overlays folder and kernel images to target/images in build folder
+do_deploy:append(){
```

Comment:
do_compile:prepend(), do_compile:append(), do_deploy:append().

Should we add machine suffix for these functions?

Reply:
I think we should add the machine-specific `rzg2l-sbc` for `do_compile:append` and `do_deploy:append` only. as `do_compile:prepend` is just for setting flags, so it can be used for other machines as well.

Updated in the commit: https://github.com/Renesas-SST/meta-renesas/commit/de7186edbd510d4722d284e38443ce313d4e89a1